### DISCUSSION TITLE
Nujik Hub onWT32-ETH01

### DISCUSSION THREAD

**Discussion by ncosta24** [technyon/nuki_hub#324] [Q&A]
Hello,

Have someone install nuki hub on a WT32-ETH01?
Need help.

**Comment by technyon** [comment]
You have to get a USB-to-serial converter and connect it to the ESP32 (RX to TX, TX to RX). Make sure to use 3.3V, you can usually set it using a jumper on the converter board. Then follow instruction in the readme for flashing. You eventually have to press one of the switches after starting the flash process in the flash tool.

If this is too complicated for you, I'd recommend to get a board with an integrated converted like the M5Stack Atom POE or one of the Olimex boards.

**Reply by ncosta24** [reply]
Thank you.
I'm trying with this one: https://www.aliexpress.com/item/1005002064355636.html
But stuck in:
![image](https://github.com/technyon/nuki_hub/assets/82029951/f77a32d0-9f89-4083-bc3a-e244d6131c0d)

**Comment by technyon** [comment]
I can only say keep trying. It took me a while to get it to flash, had to rewire it a few times. Eventually try swapping TX/RX, and make sure to press that button

**Reply by ncosta24** [reply]
You save my day!!!
swapping TX/RX was the solution!
It's working!

**Comment by technyon** [comment]
Glad to hear
